packages/checkbox/test/checkbox.test.ts

```diff
@@ -297,7 +297,6 @@ describe('Checkbox', () => {
         expect(el.indeterminate).to.be.false;
         expect(inputEl.checked).to.be.false;
         expect(inputEl.indeterminate).to.be.false;
-
```

Comment:
I don't know why some files were submitted together with my PR, they seem to be changes after linting...